src/Modal/Modal.js

```diff
@@ -34,9 +36,9 @@ const Modal = ({
         </div>
         <div className={`modal__footer modal__footer${type()}`}>
           <div className="modal__button">
-            {alert && <Button className="button button--link" onClick={() => setModalOff(true)}>Cancel</Button>}
+            {alert && <Button modifiers="link" onClick={() => setModalStatus(false)}>Cancel</Button>}
```

Comment:
I would try to do this kind of props as modifiers like enums, this will allow as to have so clear the different modifiers the component accept and also when you write the component with modifier would autocomplete it ad display you the different available options.

Would be something like that>
`
class Button extends Component() {
static linkModifier = 'link'

render() {
  return(<button modifier={props.modifier}>{children}</button>
}
}
`
And when you use this component you will do something like that:

`
  <Button modifier={Button.linkModifier}>click me<Button>
`

Also if in a future you need to change this prop for another one you just would have to change his name in the enum or static property inside the component.

Reply:
you're right